Content.Shared/_Goobstation/StationEvents/StoryPrototype.cs

```diff
@@ -0,0 +1,41 @@
+using Robust.Shared.Prototypes;
+using Content.Shared._Goobstation.StationEvents.Metric;
+
+namespace Content.Shared._Goobstation.StationEvents;
```

Comment:
Using directive is unnecessary
```suggestion
using Robust.Shared.Prototypes;

namespace Content.Shared._Goobstation.StationEvents;
```